widget/src/utils/env.ts

```diff
@@ -30,3 +30,5 @@ export const registerUriPathName = process.env.CLIENT_REGISTER_URI_PATH_NAME ||
 export const loginUriPathName = process.env.CLIENT_LOGIN_URI_PATH_NAME || "";
 
 export const forgotPasswordUriPathName = process.env.CLIENT_FORGOT_PASSWORD_URI_PATH_NAME || "";
+
+export const forceRedirect = process.env.FORCE_REDIRECT || false;
```

Comment:
echoing what @aylwin-AB mentioned.

since the value for `process.env.FORCE_REDIRECT` would be either `"true"` or `"false"`. Please confirm that this is this line is correct

Reply:
updated and tested with no assigned, false, and true value